File: BE/src/main/java/codesquad/team7/baseball/game/Team.java
package codesquad.team7.baseball.game;

import org.springframework.data.annotation.Id;

import java.util.List;

public class Team {

    @Id
    private final Long teamId;

    private final String name;
    private final List<Player> players;
    private final Integer pitcher;

    Team(Long teamId, String name, List<Player> players, Integer pitcher) {
        this.teamId = teamId;
        this.name = name;
        this.players = players;
        this.pitcher = pitcher;
    }
}

File: iOS/baseball/baseball/Model/Entity/Team.swift

import Foundation

struct Team {
    private let name: String
    private var score: Int
    private var players: Players
    private var pitcher: Pitcher
    
    init(name: String) {
        self.name = name
        self.score = 0
        self.players = Players()
        self.pitcher = Pitcher()
    }
}
